include/psp2/appmgr.h

```diff
@@ -19,8 +19,9 @@ typedef enum SceAppMgrErrorCode {
 	SCE_APPMGR_ERROR_STATE              = 0x80802013, //!< Invalid state
 	SCE_APPMGR_ERROR_NULL_POINTER       = 0x80802016, //!< NULL pointer
 	SCE_APPMGR_ERROR_INVALID            = 0x8080201A, //!< Invalid param
+	SCE_APPMGR_ERROR_TOO_LONG_ARGV      = 0x8080201D, //!< argv is too long
 	SCE_APPMGR_ERROR_INVALID_SELF_PATH  = 0x8080201E, //!< Invalid SELF path
-	SCE_APPMGR_ERROR_TOO_LONG_ARGV      = 0x8080201D  //!< argv is too long
+	SCE_APPMGR_ERROR_BGM_PORT_BUSY      = 0x80803000
```

Comment:
can you add document here?
otherwise looks good to me.

Reply:
yes, done